src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -366,6 +367,10 @@ MTR_EXTERN NSString * const MTRPreviousDataKey MTR_NEWLY_AVAILABLE;
  *                In addition to MTRDataKey, each response-value dictionary in the array may also have this key:
  *
  *                MTRPreviousDataKey : Same data-value dictionary format as the object for MTRDataKey. This is included when the previous value is known for an attribute.
+ *
+ *                The data-value dictionary also contains this key:
+ *
+ *                MTRDataVersionKey : NSNumber-wrapped uin32_t. Monotonically increaseing data version for the cluster.
```

Comment:
"increasing".

Also, "monotonically increasing" is somewhere between "misleading" and "wrong".  We should not claim it's increasing in our API documentation.  It can easily decrease.